packages/pigeon/pigeons/all_datatypes.dart

```diff
@@ -5,6 +5,19 @@
 import 'package:pigeon/pigeon.dart';
 
 class Everything {
+  Everything({
+    required this.aBool,
+    required this.anInt,
+    required this.aDouble,
+    required this.aString,
+    required this.aByteArray,
+    required this.a4ByteArray,
+    required this.a8ByteArray,
+    required this.aFloatArray,
+    required this.aList,
+    required this.aMap,
+  });
```

Comment:
Hmm, this is a problem.  I would prefer that we make the fields nullable.  That makes more sense since it matches the code that will get generated by Dart.  There is a whole different issue for making non-null types in the generated output.   We'll need to reserve what you have here for that.

Reply:
Done